pom.xml

```diff
@@ -65,6 +65,11 @@
             <groupId>org.json</groupId>
             <artifactId>json</artifactId>
         </dependency>
+        <dependency>
+            <groupId>javax.json</groupId>
+            <artifactId>javax.json-api</artifactId>
```

Comment:
It looks like we may not actually be using any classes from `javax.json`, if that's the case, let's take this back out